### PR TITLE
Stripe integration

### PR DESCRIPTION
**Pull request by SeriousMatt**

### CODE REVIEW

**Comment by SeriousMatt** [comment]
Hi @mcdurdin, sorry for the gibberish commit messages...most of these were tests to force deployments in some of the test environments. The end result is both applications are deployed to wwwroot in a front and back folder with each application running. This is also assuming the configuration variables are set correctly (see image)image. This will probably require a fresh clean workspace (as was last time) in order for it to work correctly.

**Comment by SeriousMatt** [comment]
Also, this will be deployed to the test environment. Here is my own [test environment](https://finalkeyman.azurewebsites.net/app.view.html) I have been using. Assuming this is good to go we can switch them over to live ones.

File: src/app/app.component.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export class AppComponent {
 
   constructor(private http: HttpClient) {
     const self = this;
-    let stripe_key = 'pk_test';
+    let stripe_key = 'pk_test_eM74cMSAfGXWvrLZnz7gCXuA';
```
Comment (mcdurdin) [inline src/app/app.component.ts:32]: Is this another leaked key?

**Review by mcdurdin — CHANGES_REQUESTED** [review]
There should not be keys referenced in the source code.

Can you let me know what the values should be for the environment variables you want set?
`sk_test`
`virtualDirPath`
`WEBSITE_NODE_DEFAULT_VERSION`

File: src/app/app.component.ts (hunk 1 above)
Comment (SeriousMatt) [reply src/app/app.component.ts:32]: The name of that key is Publishable key. [More here](. https://stackoverflow.com/questions/38819887/whats-the-purpose-of-the-secret-key-for-stripe-also-is-this-safe)

Comment (SeriousMatt) [reply src/app/app.component.ts:32]: Agreed. I've already set them up in the Application settings section. You should be able to view them there. When we are ready to switch over the environment to live we can then update the values.

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**